interface/reports/patient_edu_web_lookup.php

```diff
@@ -43,10 +33,9 @@
 
 <html>
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-3-2/index.js"></script>
-
+<?php Header::setupHeader(); ?>
+<!-- -->
+<title><?php echo xlt('Patient Education'); ?></title>
```

Comment:
I added this line , but this title does not go to the UI. The code on title on line 53 ( see below ) shows at the UI